packages/material-ui/src/SpeedDial/SpeedDial.test.js

```diff
@@ -125,6 +129,39 @@ describe('<SpeedDial />', () => {
     });
   });
 
+  describe('mock time', () => {
```

Comment:
We don't actually care about behavior when the time is mocked. What we want to test is the keyboard navigation when uncontrolled so that should be reflected in the test description.

Otherwise this will end up as a bin for every test using mock timers.